tests/x509/test_x509_ext.py

```diff
@@ -3429,7 +3431,9 @@ def test_excluded_none(self):
     def test_iter_input(self):
         subtrees = [x509.IPAddress(ipaddress.IPv4Network("192.168.0.0/24"))]
         nc = x509.NameConstraints(iter(subtrees), iter(subtrees))
+        assert isinstance(nc.permitted_subtrees, list)
```

Comment:
```suggestion
        assert nc.permitted_subtrees is not None
```

It's a bit cleaner to write them this way IMO.

Reply:
Done as suggested. One exception: in `test_x509_crl_builder.py` I do not:

```
        assert isinstance(ext1.value[0].full_name, list)
        uri = ext1.value[0].full_name[0]
```

as mypy complains for the second line that full_name is not indexable (Iterable is not). In other test cases, we just check the full list for equality, but here the code inspects the first element more closely.